requirements/docs.txt

```diff
@@ -65,11 +65,10 @@ sphinxcontrib-websupport==1.0.1 \
 snowballstemmer==1.2.1 \
     --hash=sha256:9f3bcd3c401c3e862ec0ebe6d2c069ebc012ce142cce209c098ccb5b09136e89 \
     --hash=sha256:919f26a68b2c17a7634da993d91339e288964f93c274f1343e3bbbe2096e1128
-idna==2.6 \
-    --hash=sha256:8c7309c718f94b3a625cb648ace320157ad16ff131ae0af362c9f21b80ef6ec4 \
-    --hash=sha256:2c6a5de3089009e3da7c5dde64a141dbc8551d5b7f6cf4ed7c2568d0cc520a8f
-certifi==0.0.8 \
-     --hash=sha256:46ecf5f7526a08cc1f8bc8232adf0cffce046f46ceff95539daec42ebc4849ef # pyup: ==0.0.8
+idna==2.5 \
+    --hash=sha256:cc19709fd6d0cbfed39ea875d29ba6d4e22c0cebc510a76d6302a28385e8bb70
+certifi==2017.4.17 \
```

Comment:
We **must not** update certifi, there's a bug with it unfortunately and it breaks certificate related stuff on production. So this should be backed out.

Reply:
I need certifi to be updated as one of the selenium test requirements needs to be on at least this version. I'll do some research.

Reply:
hmm, that's not good. Then we should **only** use a different version for the ui-tests and don't touch any of the regular requirements files. Eventually we should upgrade certifi in prod anyway but that's a hard process unfortunately.